doc/website/src/css/custom.css

```diff
@@ -22,6 +22,10 @@
   text-align: left;
 }
 
+.buttons_src-pages-index-module {
+  float: left;
```

Comment:
Center-aligned bullets in the Hero banner looked weird when I cut down the test. Left-aligning them gave a more natural grouping and separation between the title/tagline and the diagram.